templates/account/invoices/index.html

```diff
@@ -0,0 +1,70 @@
+{% extends "advantage/base_advantage.html" %}
+
+{% block title %}Invoices{% endblock %}
+{% block meta_description %}Ubuntu Advantage for Infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry, with OpenStack support, Kubernetes support included, and Livepatch, Landscape and Extended Security Maintenance to address security and compliance concerns.{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <div class="col-12">
+      <h1>Invoices</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-3">
+      <select name="marketplace" id="marketplace-dropdown">
+        <option value="">All invoices</option>
+        <option value="canonical-ua" {% if marketplace == 'canonical-ua' %}selected{% endif %}>Canonical UA</option>
+      </select>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <table>
+        <thead>
+          <th>Service</th>
+          <th>Date</th>
+          <th>Total</th>
+          <th>Download PDF</th>
+        </thead>
+        <tbody>
+          {% for invoice in invoices %}
```

Comment:
Before looping could you check for length > 0. If not can you add an empty state row saying something like no invoices available.

Reply:
@wgx What would you like to show a user with no invoices? This is what I have at the moment.

![image](https://user-images.githubusercontent.com/6387619/126630108-045dd843-292c-4267-9cba-25773b6c79ec.png)

Reply:
@albertkol this is fine

We also need a view / error (in the forthcoming acct mgmt story) for 

`You cannot view this page, please contact your account administrator for access.`

But not for now...